src/utils/intl/data/eng/index.ts

```diff
@@ -0,0 +1,48 @@
+import { WORDS } from "../constants";
+
+module.exports = {
```

Comment:
can we use i18n for translation ? It is much easier to change languages we can use this https://www.i18next.com they have a section for react & nextjs. 

We can then use service like http://phrase.com to automatically translate everything to other languages at once and it would sync back to the repo.

Reply:
You can refer to this as well https://github.com/SparkEdUAB/SparkEd/tree/master/i18n

Reply:
Noted @OlivierJM . I will do so

Reply:
By the way this can be done in a separate PR so that we first move on with this one.